### PR TITLE
--Added Next prev component

### PR DESCRIPTION
**Pull request by BaskarMitrah**
## Description

Added Next Prev component

## Related Issue

Side navbar is not working now. So I just created the design and found the previous and next page but need to check it after the completion of the side navbar

## How Has This Been Tested?

https://next-prev-spectrum-less--adobe-io-website--adobe.hlx.page/test/next-prev

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by melissag-ensemble — COMMENTED** [review]
I think we can wait for the sidebar to test this before merging. There's no rush to merge right away. @timkim your thoughts?

File: hlx_statics/blocks/next-prev/next-prev.js
Comment (melissag-ensemble) [inline hlx_statics/blocks/next-prev/next-prev.js:54]: There's a [Link component](https://github.com/adobe/adobe-io-website/blob/fresh-spectrum-less/hlx_statics/components/link.js) (based on [aio-theme's Link](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Link/index.js#L18)) that's very similar to this that's being used by [Heading](https://github.com/adobe/adobe-io-website/blob/fresh-spectrum-less/hlx_statics/components/heading.js#L39). Could we refactor this to reuse that?

In general, if there's a block that's being used by other blocks, (like Link where it's used by [Heading](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L114) and [NextPrev](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/NextPrev/index.js#L33)) I think we should create it under components to facilitate code reuse. Reason for component, rather than block is because EDS doesn't allow blocks to be nested within blocks. Please let me know what you think @timkim

Comment (timkim) [reply hlx_statics/blocks/next-prev/next-prev.js:54]: Ya, let's reuse the link component here. Should be pretty straight forward to import the link into here.

**Review by timkim — CHANGES_REQUESTED** [review]
See comment about the link component.

**Review by melissag-ensemble — APPROVED** [review]

File: hlx_statics/blocks/next-prev/next-prev.js
Comment (BaskarMitrah) [reply hlx_statics/blocks/next-prev/next-prev.js:53]: I've resolved it

**Comment by timkim** [comment]
Hmm - did something happen to the styles? The test page doesn't look right.

**Comment by BaskarMitrah** [comment]
I'll check @timkim

**Comment by BaskarMitrah** [comment]
@timkim, Actually the problem occurs in the documentation, I changed that. Please check it now

**Review by timkim — APPROVED** [review]

**Pull request merged**
